assets/prototype/domain/eventEditor/context/TicketProvider/index.tsx

```diff
@@ -15,8 +15,7 @@ const DEFAULT_CONTEXT = {
 	id: '',
 };
 
-export interface TicketContextProps extends ContextProps {}
```

Comment:
This is the same fix as here: https://github.com/eventespresso/event-espresso-core/pull/2210/commits/2740f12d791f95a766d1a3009e40f667b84118b6